CMakeLists.txt

```diff
@@ -31,7 +31,7 @@ list(APPEND CMAKE_MODULE_PATH ${CMAKE_SOURCE_DIR}/cmake
                               ${CMAKE_SOURCE_DIR}/cmake/math
 )
 include(Psi4Macros)
-no_in_source()
+include(autocmake_safeguards)
```

Comment:
Sorry, that was an incomplete back-port from libint. Add lines like https://github.com/loriab/libint1/blob/master/CMakeLists.txt#L3-L5 to the main CMakeLists.txt